src/layers/protos/FillWithOutlineProto.js

```diff
@@ -33,6 +35,16 @@ export default class FillWithOutlineProto {
     this.opacity = opacity;
     this.minzoom = minzoom;
     this.maxzoom = maxzoom;
+    console.log(id, subgroup);
+    console.log(id, floodGroup, subgroup);
+    console.log(subgroup !== undefined);
+    console.log(subgroup !== undefined && subgroup === floodGroup);
+    this.visible =
+      (subgroup !== undefined && subgroup === floodGroup) ||
+      subgroup === undefined
+        ? "visible"
+        : "none";
```

Comment:
Nit: Could you remove these consoles logs? I think this logic could also be simplified to
```suggestion
    this.visible = !subgroup || subgroup === floodGroup ? "visible" : "none";
```